### DISCUSSION TITLE
Rating Scale Enhancements

### DISCUSSION THREAD

**Discussion by ddale33** [tmrev/tmrev.io#75] [Ideas]
In the future I think that a cool scale to rate on would be rewatchability. That'd be something I would want to know if I were looking at my favorite reviewers feed. It can be obvious on extremely high and low rated movies but I think could help distinguish in the middle range.

**Comment by KegenGuyll** [comment]
@ddale33 It doesn't sound like a bad idea. My first concern would be having too many required categories when reviewing a movie and I already feel like people might be hesitate to review a movie in such detail. It would be interesting as an optional review.
